test/jellyfish_web/integration/server_socket_test.exs

```diff
@@ -151,9 +169,17 @@ defmodule JellyfishWeb.Integration.ServerSocketTest do
     assert_receive %RoomNotFound{id: ^fake_room_id}
   end
 
+  test "doesn't send messages if not subscribed", %{conn: conn} do
```

Comment:
Not sure if we need to test for this, but we could consider adding a case like "crashes if attempted to subscribe to a nonexistent event type" -- I'd for sure want to test whether subscribing to multiple different event types works, but right now we have only one...

Could potentially just add a TODO for when another event type gets added, so that we don't forget about testing this functionality

Reply:
We have a test, which checks whether sending an unexpected message causes websocket to close.
We cannot test whether subscribing to an nonexistent event causes such behavior, because this breaks on a lower level - Protobuf raises an error when trying to encode undefined event type.

Reply:
Fair enough